<launch>
	<node pkg="seabee3_demo" type="gate_demo" name="gate_demo" output="screen" args="_rate:=20" >
		<param name="forward_velocity" type="double" value="0.3" />
		<param name="forward_time" type="double" value="30.0" />
		<param name="depth" type="double" value="0.4" />
		<param name="dive_time" type="double" value="5.0" />
	</node>
</launch>
